exercicios/para-casa/entregas/amanda-bittencourt/SavingAccount.js

```diff
@@ -0,0 +1,51 @@
+import { BanksAccount } from './BanksAccount.js';
+
+export class SavingAccount extends BanksAccount {
+  incomeRate; 
+  incomeDay;
+  #qtdWithdrawal = 0;
+  #MAX_OF_WITHDRAWAL = 2;
+  #withdrawalTax = 0.03;
+
+  constructor(client, bank, accountNumber, agencyNumber, incomeRate, incomeDay) {
+    super(client, bank, accountNumber, agencyNumber) 
+    this.incomeRate = incomeRate; 
+    this.incomeDay = incomeDay; 
+  }
+
+  cashWithdrawal(amount) {
+    console.log(`Você já realizou ${this.#qtdWithdrawal} retiradas. Você tem direito a ${this.#MAX_OF_WITHDRAWAL} retiradas gratuitas.`)
+
+    if(this.#qtdWithdrawal < this.#MAX_OF_WITHDRAWAL) {
+      if(this.balance >= amount) {
+        this.balance -= amount;
+        this.#qtdWithdrawal++;
+        console.log(`Retirada realizada com sucesso. Seu saldo restante é R$ ${this.balance},00`)
```

Comment:
console.log(`Retirada realizada com sucesso. Seu saldo restante é R$ ${this.balance.toFixed(2)}`);